lift/Readme.md

```diff
@@ -98,8 +97,7 @@ This is error-prone and stressful, especially when you're operating on your prod
 
 ## The Lift Approach
 
-Prisma's Lift works differently. While Lift still has a `migrations/` folder, the migrations are generated for you. With Lift, you just need to change your
-`schema.prisma` file and run `lift save`. This will generate the necessary steps to transition your schema from A to B.
+Prisma's Lift works differently. While Lift still has a `migrations/` folder, the migrations are generated for you. With Lift, you just need to change your project's `schema.prisma` file and run `lift save`. This will generate the necessary steps to transition your schema from A to B.
 
 A result might look like this:
```

Comment:
The result of what exactly is this actually @matthewmueller? The previous sentence makes you think these could be the generated steps, but I don't think that is the case, right?

Reply:
> The result of what exactly is this actually @matthewmueller?

I don't know what this means haha.

The steps are generated for you, sometimes you need to edit them by hand though (renaming a model).

Reply:
So what do https://github.com/prisma/specs/pull/195/files#diff-8302b741093dd01fdda810da584dac22L106-L130 actually show here? What is this - it looks like a diff of a model part of a schema. But why is this here? How does it relate to Lift?

Reply:
Ping @matthewmueller